pages/index.tsx

```diff
@@ -141,48 +141,57 @@ export default function Home() {
             <Button type="submit">Lisää</Button>
           </form>
         </div>
-        <div className="mt-3">
+        <div className="pl-8 pr-8 mt-3 w-full sm:w-[20rem]">
```

Comment:
Yleensä on hyvä käyttää valmiita kokoja, esim. tuon sun `sm:w-[20rem]` voi korvata `sm:max-w-xs`. Mutta nyt myös isoilla näytöillä 20 rem on maksimikoko -> muuttaisin niin että isoilla näytöillä sisältöä näkyy enemmän. Itse asiassa pienillä näytöillä kokoa ei mun mielestä tarvitse rajoittaa, vaan on hyvä käyttää koko näytön tila. Eli itse laittaisin koon seuraavasti:
```suggestion
        <div className="pl-8 pr-8 mt-3 w-full max-w-xl">
```